sources/web/datalab/static/datalab.js

```diff
@@ -240,7 +240,7 @@ function initializePage(dialog, saveFn) {
         path = path.substr('/notebooks'.length);
         path = path.substr(0, path.lastIndexOf('/'));
       }
-      path = encodeURIComponent('/content' + path);
+      path = '/content' + path;
```

Comment:
It's actually resulting in the wrong string in this case, where ungit expects an unencoded path.